tests/end_to_end/server.py

```diff
@@ -156,16 +164,24 @@ class Ga4ghServerForTesting(ServerForTesting):
     """
     A ga4gh test server
     """
-    def __init__(self):
-        super(Ga4ghServerForTesting, self).__init__(ga4ghPort)
+    def __init__(self, useOidc=False):
+        protocol = 'https' if useOidc else 'http'
+        super(Ga4ghServerForTesting, self).__init__(ga4ghPort, protocol)
         self.configFile = None
+        self.useOidc = useOidc
 
     def getConfig(self):
         config = """
 SIMULATED_BACKEND_NUM_VARIANT_SETS = 10
 SIMULATED_BACKEND_VARIANT_DENSITY = 1
 DATA_SOURCE = "__SIMULATED__"
-DEBUG = True"""
+DEBUG = True
+"""
+        if self.useOidc:
+            config += """
+TESTING = True
+OIDC_PROVIDER = "https://localhost:8443"
```

Comment:
port should not be hardcoded; use string formatting to substitute in the `oidcOpPort` variable